Formula/fasttree.rb

```diff
@@ -16,20 +17,24 @@ class Fasttree < Formula
   # http://www.microbesonline.org/fasttree/#BranchLen
   # http://darlinglab.org/blog/2015/03/23/not-so-fast-fasttree.html
 
-  option "without-double", "Disable double precision floating point. Use single precision floating point & enable SSE"
-  option "without-openmp", "Disable multithreading support"
-  option "without-sse", "Disable SSE parallel instructions"
-
-  if build.with? "openmp"
-    fails_with :clang # needs openmp
-    depends_on "gcc" if OS.mac? # needs openmp
+  on_macos do
+    depends_on "libomp"
   end
 
   def install
-    opts = %w[-O3 -finline-functions -funroll-loops]
-    opts << "-DOPENMP" << "-fopenmp" if build.with? "openmp"
-    opts << "-DUSE_DOUBLE" if build.with? "double"
-    opts << "-DNO_SSE" if build.without? "sse"
+    opts = %w[
+      -O3
+      -finline-functions
+      -funroll-loops
+      -DOPENMP
+      -DUSE_DOUBLE
+      -DNO_SSE
```

Comment:
```ruby
opts << "-DNO_SSE" if build.without? "sse"
```

```suggestion
```

`-DNO_SSE` was not used by default previously

Reply:
Oh whoops

Reply:
This comment still applies.

Reply:
Strange I was sure I had made this change